.golangci.ci.yaml

```diff
@@ -28,3 +29,9 @@ issues:
     - linters:
         - gosec
       text: "G(101|107|204|306|402)"
+    - linters:
+        - staticcheck
+      text: "SA(1002|1006|4000|4006)"
+    - linters:
+        - staticcheck
+      text: "(ioutil|KubeCtl|NewCertManagerBasicCertificate|DeprecatedCertificateTemplateFromCertificateRequestAndAllowInsecureCSRUsageDefinition|eab.KeyAlgorithm|testCA.Subjects|RootCAs.Subjects|pki.GenerateTemplate|adal.NewServicePrincipalTokenFromFederatedToken|azure-sdk-for-go|c.SingleInflight|x509.ParseCRL|v.client.RawRequest)"
```

Comment:
I'll deal with these in separate PRs. 
`NewCertManagerBasicCertificate` and `NewCertManagerBasicCertificateRequest` are used all over the tests and will take some time to replace.